ProjectSystem.sln

```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.28301.62
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ProjectSystemSetup", "src\ProjectSystemSetup\ProjectSystemSetup.csproj", "{9FBD7EF2-9449-486D-9FDD-FA56160AA8BB}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Setup", "Setup", "{AC8DB8AE-AC9F-4503-83C6-255B66C318C6}"
```

Comment:
This was deliberately moved to the top of the solution; when you first open the solution, the default F5 project is based on the first project in the solution based on a depth first walk of the "nested" folders.

Reply:
TIL 👍